airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/ConfigWriter.java

```diff
@@ -29,6 +41,36 @@
 @SuppressWarnings("PMD.CognitiveComplexity")
 public class ConfigWriter {
 
+  /**
+   * @return A set of connectors (both source and destination) that are already used in standard
+   *         syncs. We identify connectors by its repository name instead of definition id because
+   *         connectors can be added manually by users, and their config ids are not always the same
+   *         as those in the seed.
+   */
+  static Set<String> getConnectorRepositoriesInUse(final DSLContext ctx) {
+    return getActorDefinitionsInUse(ctx)
+        .map(Record4::value2)
```

Comment:
can we use `map(record -> record.get(ACTOR_DEFINITION.DOCKER_REPOSITORY))` here? This would help to understand what we are fetching here. (Same bellow)

Reply:
Much better indeed. done.